samples: tfm: Enable secp256r1 for tfm_psa_template (#11016)
nrfconnect/sdk-nrf: samples/tfm/tfm_psa_template/README.rst

This sample supports firmware update of both the application and TF-M, and the second stage bootloader.

The firmware update process requires signature verification keys in order to sign the images used in the process.
The nRF Secure Immutable bootloader and MCUboot will use signing keys that should not be used in production.

> For signing and verification of images, use ECDSA with secp256r1-sha256 that is supported by the nRF cryptographic libraries :ref:`nrf_oberon_readme` and :ref:`crypto_api_nrf_cc310_bl`.

Author: Yeap, it's more common to see it lower case. There is not a wrong way of writing it as far as I know.

> For signing and verification of images, use ECDSA with secp256r1-sha256 that is supported by the nRF cryptographic libraries :ref:`nrf_oberon_readme` and :ref:`crypto_api_nrf_cc310_bl`.

Reviewer: ```suggestion
For signing and verifying images, use ECDSA with secp256r1-sha256, which is supported by the |NCS| cryptographic libraries :ref:`nrf_oberon_readme` and :ref:`crypto_api_nrf_cc310_bl`.
```

> The firmware update process requires signature verification keys in order to sign the images used in the firmware update process.

Reviewer 2: Could we say the following instead?
The firmware update process requires signature verification keys in order to sign the images used in the process.

Author: Sure

The nRF Secure Immutable bootloader and MCUboot will use signing keys that should not be used in production.

> Use SECP256-SHA256 for signing and verification of the images, since it is supported by the nRF cryptographic libraries :ref:`nrf_oberon_readme` and :ref:`crypto_api_nrf_cc310_bl`.

Reviewer 2: How about:
```suggestion
For signing and verification of images, use SECP256-SHA256 that is supported by the nRF cryptographic libraries :ref:`nrf_oberon_readme` and :ref:`crypto_api_nrf_cc310_bl`.
```

> We recommend using SECP256-SHA256 for signing/verification of the images since it is supported by our cryptographic libraries :ref:`nrf_oberon_readme` and :ref:`crypto_api_nrf_cc310_bl`.

Reviewer: ```suggestion
Use SECP256-SHA256 for signing and verification of the images, since it is supported by the nRF cryptographic libraries :ref:`nrf_oberon_readme` and :ref:`crypto_api_nrf_cc310_bl`.
```

> Example on how to generate and use keys:

Reviewer 2: Could we change this to:
Below is an example of how to generate and use the keys.

Author: Sure
